assets/src/components/Label.js

```diff
@@ -1,8 +1,28 @@
 import React from 'react'
 
+const alignThreshold = 25
+
+const getLeft = (value) => {
```

Comment:
NipPick: Can the name be getLeftAlignLen()? and same with getRighAlignLen()

Reply:
They are literally getting the values for the left and right properties.